### PR TITLE
Adds Yanna Trevidic's custom item

### PR DESCRIPTION
**Pull request by feartheblackout**
adds the custom item approved hither: https://forums.aurorastation.org/topic/17006-yanna-trevidics-techno-conglomerate-cr-dress

definitely shitcode but i was at least able to get it working whenever it can be reviewed :)

### CODE REVIEW

**Comment by feartheblackout** [comment]
!review

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -2223,3 +2223,50 @@ All custom items with worn sprites must follow the contained sprite system: http
...
+	item_state = "yanna_dress"
+	contained_sprite = TRUE
+	var/list/emotional_settings = list("0", "1", "2", "3")
+	var/emotional_choice
+	action_button_name = "Adjust Emotional Setting"
+
+/obj/item/clothing/under/fluff/yanna_dress/Initialize()
+	for(var/setting in emotional_settings)
+		emotional_settings[setting] = image('icons/obj/custom_items/yanna_dress.dmi', "level_[emotional_settings.Find(setting) - 1]")
+	.=..()
```
Comment (NonQueueingMatt) [inline code/modules/customitems/item_defines.dm:2243]:
```suggestion
	. = ..()
```

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -2223,3 +2223,50 @@ All custom items with worn sprites must follow the contained sprite system: http
...
+
+/obj/item/clothing/under/fluff/yanna_dress/attack_self(mob/user)
+	choose_setting(user)
+
+/obj/item/clothing/under/fluff/yanna_dress/verb/adjust_setting()
+	set category = "Object"
+	set name = "Adjust Emotional Setting"
+	set src in usr
+
+	choose_setting(usr)
```
Comment (NonQueueingMatt) [inline code/modules/customitems/item_defines.dm:2253]: Add an use_check_and_message here.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
whoops was not meant to be an approval

**Comment by mikomyazaki** [comment]
The icon file appears to contain one blank icon.

**Review by mikomyazaki — CHANGES_REQUESTED** [review]
The icon file appears to be blank.

**Comment by feartheblackout** [comment]
oookay no idea why that keeps happening

**Comment by feartheblackout** [comment]
haven't forgotten about this. just haven't been motivated to readd the icons after losing them twice sorry

**Comment by feartheblackout** [comment]
sorry for the wait, ready for review now

**Review by NonQueueingMatt — APPROVED** [review]

**Review by mikomyazaki — APPROVED** [review]

**Pull request merged**